specs/experimental/custom-gas-token.md

```diff
@@ -0,0 +1,471 @@
+# Custom Gas Token
```

Comment:
Did we ever made a decision around calling this Custom Native Asset instead? I know @maurelian suggested this, and I do agree with it, but don't have a strong opinion. Probably best to go with whatever we think is more intuitive to users/devs

Reply:
I lean towards renaming it as well but we were unable to get complete consensus on it so its staying as is

Reply:
leaving this open until there's a conclusion

Reply:
I am going to close this, if we want to revisit this we can bring it up on slack